### PR TITLE
Added Custom Server prefixes

### PR DESCRIPTION
**Pull request by nemesis0606**
Now, the bot can have different prefixes in different servers, i.e the server owners / admins can set a different prefix than the bot's default one.
 
A glimpse of the new feature :
prefix

### CODE REVIEW

File: src/main.py [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
 intents = discord.Intents.default()
 intents.members = True
 
-# Create instance of a bot object
-client = commands.Bot(command_prefix = '-', intents = intents)
+# Function that checks for prefix for that specific guild from a set of guilds
+def get_prefix(client, message):
+    with open ("prefixes.json", "r") as f:
+        prefix = json.load(f)
+    return prefix[str(message.guild.id)]
```
Comment (Dsantra92) [inline src/main.py:16]: Any specific reason that we are using json?

Comment (sdk2k01) [reply src/main.py:16]: Yeah, we need to store the list of servers as keys and the respective prefixes as values which need to be loaded on calling the bot instance. Once the file is opened, the function returns the prefix for that server in the `commands.Bot(command_prefix = get_prefix, ...)` instance and it is done for each and every server the bot is operating in.

Comment (Dsantra92) [reply src/main.py:16]: Ah, yes!!

**Comment by Dsantra92** [comment]
Well done @nemesis0606!!

**Pull request merged**